GUIDE.md

```diff
@@ -71,7 +71,17 @@ Use a web browser to manage the simulated Thread network:
 
 See [OTNS CLI Reference](cli/README.md). 
 
-### OTNS Python scripting
+## OTNS Python Scripting
 
-[pylibs/otns](pylibs/otns) provides utilities to create and manage simulations through OTNS CLI. 
-Check the test scripts in [pylibs/examples](pylibs/examples) for examples of how to use otnscli.py. 
+[pylibs/otns library](pylibs/otns) provides utilities to create and manage simulations through OTNS CLI. 
+
+**Check `pylibs/otns` document**
+1. Start `pydoc3` document server:
+    ```bash
+    pydoc3 -p 8080
+    ```
+2. Open a web browser and navigate to http://localhost:8080/otns.html.
+
+**Check examples:**
+
+Check the scripts in [pylibs/examples](pylibs/examples) for examples.
```

Comment:
I would get rid of the **Check examples:** line and just say:

For example test scripts, see [pylibs/examples](pylibs/examples).